London class 8/simon lawrence

SimonLawrenceGB opened the pull request.

sinopungula reviewed (commented):
reviewed your work, good job.

RodrigueNdzana commented:
Well done Brett

JarrodBen commented:
Hey Brett, I think it would be best to have multiple commits by either uploading after each exercise folder is done or the exercise folder and then the mandatory, it will be easier for reviewers to check your work at a certain point, understandable if you don't have access to the internet all the time, but it's just something to keep in mind.

SimonLawrenceGB committed:
A-undefined

exercise.js complete.

SimonLawrenceGB committed:
Exercise B, C, D

Completed with the help of homework club but not fully understood. More assistance needed.

SimonLawrenceGB committed:
Exercises B, C, D

Improved and updated.

SimonLawrenceGB committed:
E-for-loop

exercise1.js complete.

SimonLawrenceGB committed:
E-for-loop

exercise2.js complete.

SimonLawrenceGB committed:
F-for-of-loop

exercise.js complete.

SimonLawrenceGB committed:
2-mandatory

1-weather-report.js failed attempt.

SimonLawrenceGB committed:
2-mandatory

1-weather-report.js,
2-retrying-random-numbers,
Complete
(tests failed because I forgot to install and run npm tests)

SimonLawrenceGB committed:
2-mandatory

3-financial-times.js complete.

SimonLawrenceGB committed:
2-mandatory

4-stocks.js complete.

SimonLawrenceGB committed:
2-mandatory / 3-financial-times.js

Name corrections.

SimonLawrenceGB committed:
Small code format changes and comments
